jobs/config.json

```diff
@@ -92,6 +92,7 @@
       "--check-leaked-resources",
```

Comment:
we should move all of this to another owned directory with you and yuju as OWNERS :-)